{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "de750854_1082defe",
        "filename": "/COMMIT_MSG",
        "patchSetId": 11
      },
      "lineNbr": 26,
      "author": {
        "id": 13315
      },
      "writtenOn": "2023-04-10T14:47:01Z",
      "side": 1,
      "message": "Please write \"Fixes #58785\" (literally). Thanks.",
      "range": {
        "startLine": 25,
        "startChar": 0,
        "endLine": 26,
        "endChar": 41
      },
      "revId": "86cdbac52fa1922128e5e8d41f5a6b4f7a3f2aa5",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "83af3055_8d6bc03c",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 11
      },
      "lineNbr": 0,
      "author": {
        "id": 61503
      },
      "writtenOn": "2023-04-10T07:20:48Z",
      "side": 1,
      "message": "@bcmills@google.com and @cherryyz@google.com Can you please run the TryBot on s390x platform.\n\nTRY\u003dlinux-s390x-ibm",
      "revId": "86cdbac52fa1922128e5e8d41f5a6b4f7a3f2aa5",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705"
    }
  ]
}